### ISSUE TITLE
Iso Date Format

### ISSUE THREAD

**Issue by gregoryagu** [aurelia-ui-toolkits/aurelia-kendoui-bridge#627]
I am using a SqlServer DateTimeOffset field.

The date is converted to a string like this: 
2016-09-01T04:00:00+00:00

The kendoParaseDate value converter converts to this string:
Mon May 16 2016 00:00:00 GMT-0400 (Eastern Daylight Time)

However, when I try to format like this:

${currentItem.startDate | kendoParseDate:'MM/dd/yyyy'}

It produces:
Thu Sep 01 2016 00:00:00 GMT-0400 (Eastern Daylight Time)

In other words, the format is not applied.

I have stepped through the code and the format string is passed to kendo, so it may be an issue in the underlying component.

**Comment by gregoryagu** [comment]
I am going to close this, as it is not an issue with aurelia-kendoui-bridge.
